### PR TITLE
Azure Stack Fabric Specification Rewrite

### PR DESCRIPTION
**Pull request by deathly809**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

@sergey-shandar 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by sergey-shandar** [comment]
@jhendrixMSFT Hi, could I take the PR?

**Comment by sergey-shandar** [comment]
https://github.com/Azure/azure-rest-api-specs/pull/2809

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/stable/2016-05-01/Operations.json [hunk 1]
```diff
@@ -1,175 +0,0 @@
-{
-    "swagger": "2.0",
```
Comment (sergey-shandar) [inline specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/stable/2016-05-01/Operations.json]: Why this file is removed?

**Review by sergey-shandar — CHANGES_REQUESTED** [review]

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/stable/2016-05-01/Operations.json (hunk 1 above)
Comment (deathly809) [reply specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/stable/2016-05-01/Operations.json]: This is where the long running operations poll.  They have no way to call it unless they do some packet sniffing to get the operation id.

**Comment by deathly809** [comment]
Moved to preview.

**Review by sergey-shandar — APPROVED** [review]

**Pull request merged**